library/core/src/ptr/mut_ptr.rs

```diff
@@ -179,6 +188,41 @@ impl<T: ?Sized> *mut T {
         self as usize
     }
 
+    /// Gets the "address" portion of the pointer, and 'exposes' the "provenance" part for future
+    /// use in pointer-to-integer casts.
```

Comment:
Shouldn't this say "integer-to-pointer"?

Reply:
Oh yes it should. :)

Reply:
Actually it should probably just link to `from_exposed_addr`, since we consider the casts soft-deprecated if people are using these APIs.